Fix bezier size change in graphviz

In graphviz this commit was applied in 10.0.1
git describe --contains

ebc8c1a25c4aa4385
API BREAK: make 'bezier.size' a 'size_t'

This commit fixes the breakage.


---
 libyui-qt-graph/src/QY2Graph.cc |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/libyui-qt-graph/src/QY2Graph.cc
+++ b/libyui-qt-graph/src/QY2Graph.cc
@@ -188,7 +188,7 @@
 {
     QPainterPath path;
     path.moveTo(gToQ(bezier.list[0]));
-    for (int i = 1; i < bezier.size - 1; i += 3)
+    for (size_t i = 1; i < bezier.size - 1; i += 3)
 	path.cubicTo(gToQ(bezier.list[i]), gToQ(bezier.list[i+1]), gToQ(bezier.list[i+2]));
     return path;
 }
@@ -273,7 +273,7 @@
 
     if (poly->peripheries != 1)
     {
-	qWarning("unsupported number of peripheries %d", poly->peripheries);
+	qWarning("unsupported number of peripheries %zu", poly->peripheries);
     }
 
     const int sides = poly->sides;
@@ -406,7 +406,7 @@
 	    if (spl == NULL)
 		continue;
 
-	    for (int i = 0; i < spl->size; ++i)
+	    for (size_t i = 0; i < spl->size; ++i)
 	    {
 		const bezier& bz = spl->list[i];
 
